tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -795,9 +801,49 @@ def _post_fit(self, keep_only_best=False, refit_full=False, set_best_to_refit_fu
         if keep_only_best:
             self.delete_models(models_to_keep='best', dry_run=False)
 
+        if calibrate and self.problem_type in PROBLEM_TYPES_CLASSIFICATION:
+            self._calibrate_model()
+        else:
+            logger.log(30, 'WARNING: calibrate is only applicable to classification problems')
+
         if save_space:
             self.save_space()
 
+    def _calibrate_model(self, model_name: str = None, lr: float = 0.01, max_iter: int = 1000, init_val: float = 1.0):
+        """
+        Applies temperature scaling to the best autogluon model. Applies
+        inverse softmax to predicted probs then trains temperature scalar
+        on validation data to maximize negative log likelihood. Inversed
+        softmaxes are divided by temperature scalar then softmaxed to return
+        predicted probs
+
+        Parameters:
+        -----------
+        model_name: str: default=None
+            model name to retrieve from trainer and fit
```

Comment:
`predictor` level documentation should not mention trainer, assume users do not know what trainer/learner are.